Gemfile

```diff
@@ -1,6 +1,9 @@
 source 'https://rubygems.org'
 
-gem 'iban-tools', github: 'opahk/iban-tools', branch: 'banking_data'
+# a gem's github dependencies have to be repeated in gemfiles of apps that use
+# this gem
+gem 'iban-tools', github: 'gapfish/iban-tools'
+gem 'banking_data', github: 'gapfish/banking_data' # dependency of iban-tools
```

Comment:
Nice-to-have: the comment makes it sound as if the line is superfluous. Is it?

Reply:
@trebor8x It's supposed to say the opposite. is it better now?

iban-tools needs to list banking_data
iban-validation needs to list iban-tools and banking_data
gapfish mothership needs to list iban-validation, iban-tools and banking_data